### PR TITLE
Feat/1655 dark theme support for code block

### LINKED ISSUE #1655: Dark theme support for code block

### PR DESCRIPTION
**Pull request by frolyo**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts [hunk 1]
```diff
@@ -47,3 +47,16 @@ export function highlightCode(pre: HTMLPreElement, languageName: string, code: s
...
+    if (themeInfo.currentTheme === 'light') {
+        // @ts-ignore
+        void import('highlight.js/styles/intellij-light.css');
+    } else {
+        // @ts-ignore
+        void import('highlight.js/styles/atom-one-dark.css');
+    }
+}
+
+applyTheme(Theme.info);
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts]: When this code will be invoked?

File: src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts [hunk 2]
```diff
@@ -47,3 +47,16 @@ export function highlightCode(pre: HTMLPreElement, languageName: string, code: s
         errorLog?.log(`highlightCode: failed to highlight code`, e);
     }
 }
+
+function applyTheme(themeInfo: ThemeInfo){
+    if (themeInfo.currentTheme === 'light') {
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts:37]: Does importing theme css replaces the old one, right?

File: src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ import c from 'highlight.js/lib/languages/c';
 import cpp from 'highlight.js/lib/languages/cpp';
 import csharp from 'highlight.js/lib/languages/csharp';
 import { Log } from 'logging';
-import 'highlight.js/styles/intellij-light.css'
+import { Theme, ThemeInfo } from 'theme';
 
 const { errorLog } = Log.get('CodeBlockMarkupView');
 hljs.registerLanguage('bash', bash);
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts]: I am curious if the code affects startup time?

File: src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts (hunk 2 above)
Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts:37]: on import

File: src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts (hunk 1 above)
Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts]: on import

File: src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts (hunk 3 above)
Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts]: suggestions?

Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts]: I guess it's not a heavy operation. If we have kind of tracer on client side, we can test it

File: src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts (hunk 2 above)
Comment (crui3er) [reply src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts:37]: What do mean 'on import'?

Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts:37]: sorry, was answering on previous question

Comment (frolyo) [reply src/dotnet/Chat.UI.Blazor/Components/MarkupParts/CodeBlockMarkupView/code-block-markup-view.ts:37]: yes

**Pull request merged**